jerry-ext/extfunc/extfunc-assert.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/extfunc.h"
```

Comment:
The 'ext' in 'jerryscript-ext' stands for 'extension' (and there, we've concluded to use 'ext', not 'util'). The 'ext' in 'extfunc' stands for 'external', as in 'jerry_create_external_function' (and there, we haven't concluded what would be the best name -- I've copied this over from the original PR as I had to give some name to the jerry-ext module).